infrastructure/restapi/src/testFixtures/java/tech/pegasys/teku/infrastructure/restapi/OpenApiTestUtil.java

```diff
@@ -82,6 +83,36 @@ public void compareToKnownDefinitions(
     checkAllExpectedFilesExist(tempDir, path);
   }
 
+  @SuppressWarnings("rawtypes")
+  public void compareToKnownDefinition(final Class clazz) throws IOException {
+    final Path tempDir = Files.createTempDirectory("eventTypeDefinitions");
+    final String filename = clazz.getSimpleName() + ".json";
+    final String namespace = "schema";
+
+    final SchemaFactoryWrapper visitor = new SchemaFactoryWrapper();
+
+    mapper.acceptJsonFormatVisitor(mapper.constructType(clazz), visitor);
+    final JsonNode node =
+        mapper.readTree(
+            mapper.writerWithDefaultPrettyPrinter().writeValueAsString(visitor.finalSchema()));
+    Files.write(tempDir.resolve(filename), prettyJson(node).getBytes(UTF_8));
```

Comment:
Do we need to write these to file all the time? I can see it would be useful when initially creating them to have them all written but probably not necessary every time we run the test.

Reply:
fair point, will see if we can get away without doing that, I don't use it when initially creating the test either...